MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -1377,7 +1378,7 @@ static MTLVertexFormat mvkAdjustFormatVectorToSize(MTLVertexFormat format, uint3
 			maxBinding = max(pVKVB->binding, maxBinding);
 			uint32_t vbIdx = getMetalBufferIndexForVertexAttributeBinding(pVKVB->binding);
 			auto vbDesc = inputDesc.layouts[vbIdx];
-			if (pVKVB->stride == 0) {
+			if (isVtxStrideStatic && pVKVB->stride == 0) {
```

Comment:
On line 1373 there should probably be another check for this. (I can't actually annotate 1373 with this comment.)

Reply:
Ah, yes. Thanks. Good catch. Fixed now.

Reply:
Ah yes. Thanks. Good catch. Fixed now.